Resources/Maps/hive.yml

```diff
@@ -63741,6 +63741,9 @@ entities:
     - type: Transform
       pos: 144.5,4.5
       parent: 1
+    - type: SingletonDeviceNetServer
+      active: False
```

Comment:
how does disabling it fix it not working what

Reply:
> how does disabling it fix it not working what

No idea. But all the other maps have that comp, including upstream. But for some reason when you map a new one, the comp isn't being added and it doesn't work.

Reply:
See also: [#1346](https://github.com/DeltaV-Station/Delta-v/issues/1346)